src/Qmmands/Results/Failed/CommandNotFoundResult.cs

```diff
@@ -5,7 +5,9 @@
     /// </summary>
     public sealed class CommandNotFoundResult : FailedResult
     {
-        /// <inheritdoc />
+        /// <summary>
```

Comment:
Why have you moved the documentation down from `FailedResult` into individual child classes?

Reply:
IntelliSense doesn't support it.